packages/flame/lib/src/experimental/max_viewport.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:ui';
+
+import 'package:vector_math/vector_math_64.dart';
+
+import 'viewport.dart';
+
+/// The default viewport, which is as big as the game canvas allows.
+///
+/// This viewport does not perform any clipping. (?)
```

Comment:
```suggestion
/// This viewport does not perform any clipping.
```
I think that is totally correct given the impl & desc